### ISSUE TITLE
NuGet package questions

### ISSUE THREAD

**Issue by tpluscode** [GitTools/GitLink#45]
First, in issue #44 I mentioned that it's not possible to install the package in projects targeting .NET 4.0 and earlier. Looking at the package I realized that it actually adds a reference to the project where it's installed.

Is that really necessary? I figure that GitLink is more related to the general build process and as such I think that the exe could be directly in the `content`. To actually use it's necessary to add a call to the build script manually anyway.

**Comment by GeertvanHorrik** [comment]
Agree. How would you change this. To add it as a tool? Interested in doing a PR for this?
